torch/csrc/jit/runtime/register_special_ops.cpp

```diff
@@ -372,6 +372,20 @@ RegisterOperators reg({
         TORCH_SELECTIVE_SCHEMA("aten::is_scripting() -> bool"),
         [](Stack* stack) { push(stack, true); },
         aliasAnalysisFromSchema()),
+    OperatorGenerator(
```

Comment:
These schemas look redundant to me (could be all covered by the variadic version), but I'm assuming this is because of some implementation detail on the Python side?

Reply:
Nope, you're absolutely correct. The variadic schema is sufficient. Thanks!